bindata/manifests/operator-webhook/server.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: operator-webhook
+  namespace: {{.Namespace}}
+  labels:
+    app: operator-webhook
+  annotations:
+    kubernetes.io/description: |
+      This deployment launches the sriov network operator admission control webhook component.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: operator-webhook
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: operator-webhook
+    spec:
+      serviceAccountName: operator-webhook-sa
+      nodeSelector:
+        beta.kubernetes.io/os: linux
```

Comment:
nit: this is now `kubernetes.io/os`

Reply:
As label `beta.kubernetes.io/os` is also used in 4.3, we decide to not change it for this release.